homeassistant/components/vasttrafik/sensor.py

```diff
@@ -107,7 +112,7 @@ def update(self):
             self._departureboard = self._planner.departureboard(
                 self._departure['id'],
                 direction=self._heading['id'] if self._heading else None,
-                date=datetime.now()+self._delay)
+                date=datetime.now(pytz.timezone(self.timezone))+self._delay)
```

Comment:
I am very confused. You are first converting our timezone to a string, then store it, then convert it back ? Why not just access `self.hass.config.time_zone` here ?

Reply:
It has to be a string, but I have simplified it by using `self.hass.config.time_zone` directly as you suggested.